backend/inspirehep/logger/ext.py

```diff
@@ -17,6 +18,22 @@
 LOGGER = structlog.getLogger()
 
 
+def log_request_info(*args, **kwargs):
+    structlog.threadlocal.bind_threadlocal(request_path=request.path)
+
+
+def remove_request_logging(*args, **kwargs):
+    structlog.threadlocal.clear_threadlocal()
+
+
+def url_rule(request):
+    # Pretty-printing adapted from werkzeug.routing.Route.__repr__
+    return "".join(
+        f"<{data}>" if is_dynamic else data
+        for (is_dynamic, data) in request.url_rule._trace
+    ).lstrip("|")
```

Comment:
this uses the private `_trace` property, otherwise you get ugly stuff like the value in https://github.com/inspirehep/inspirehep/blob/611756c422e5c906d87ab0abd19b0fa521aa3d19/backend/inspirehep/records/config.py#L94